services/backend/src/service/connectivity.js

```diff
@@ -113,6 +123,41 @@ module.exports = () => {
 
       return data;
     },
+
+    /**
+     * Get the layer of a estrategic ecosystem in a given area
+     *
+     * @param {String | Number} areaType area type
```

Comment:
Check again the param types of areaType and areaId

Reply:
Done